README.md

```diff
@@ -66,13 +69,87 @@ findPorts({ firefoxOSSimulator: true }).then(function(results) {
 }, function(err) {
   console.log(err);
 });
+```
+
+The output from the above code might look like the following:
+```javascript
+[ { type: 'b2g', port: 56567, pid: 45876 },
+  { type: 'firefox', port: 6000, pid: 3718 },
+  { type: 'device', port: 8001, deviceId: '3739ced5' } ]
+```
 
+Including the `detailed` option interrogates each runtime for additional information:
```

Comment:
This phrasing would probably confuse a non-native speaker. Even I was a bit thrown by it.

"Use the `detailed` option for additional information:"

is probably fine.